src/lib/db/environment-store.ts

```diff
@@ -201,6 +202,11 @@ export default class EnvironmentStore implements IEnvironmentStore {
                 { column: 'created_at', order: 'asc' },
             ]);
         const rows = await qB;
+
+        if (query) {
```

Comment:
This seems wrong. Await is already executing the query. Where statement should be before await.